tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -80,42 +80,7 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc patch openstackcontrolplane openstack --type=merge --patch '
-    spec:
-      glance:
-        enabled: true
-        template:
-          databaseInstance: openstack
-          customServiceConfig: |
-                [DEFAULT]
-                enabled_backends = default_backend:swift
-                [glance_store]
-                default_backend = default_backend
-                [default_backend]
-                swift_store_create_container_on_put = True
-                swift_store_auth_version = 3
-                swift_store_auth_address = {{ .KeystoneInternalURL }}
-                swift_store_endpoint_type = internalURL
-                swift_store_user = service:glance
-                swift_store_key = {{ .ServicePassword }}
-          storageClass: "local-storage"
-          storageRequest: 10G
-          glanceAPIs:
-            default:
-              replicas: 1
-              override:
-                service:
-                  internal:
-                    metadata:
-                      annotations:
-                        metallb.universe.tf/address-pool: internalapi
-                        metallb.universe.tf/allow-shared-ip: internalapi
-                        metallb.universe.tf/loadBalancerIPs: 172.17.0.80
-                  spec:
-                    type: LoadBalancer
-              networkAttachments:
-              - storage
-    '
+    oc patch openstackcontrolplane openstack --type=merge --patch-file={{ lookup('ansible.builtin.file', 'glance_swift.yaml') }}
```

Comment:
--patch-file now has the parsed content of glance_swift.yaml, but not the filename of the parsed content file

Reply:
right, I was wrong about using the `lookup` filter. I'm going to update this line with the new patchset and try to rely on the special, builtin variable `role_path` [1].

[1] https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html#term-role_path